pkg/gardenlet/controller/shoot/state/reconciler.go

```diff
@@ -55,5 +66,59 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		return reconcile.Result{}, fmt.Errorf("error retrieving object from store: %w", err)
 	}
 
-	return reconcile.Result{}, nil
+	// if shoot got deleted or is no longer managed by this gardenlet (e.g., due to migration to another seed) then don't requeue
+	if shoot.DeletionTimestamp != nil || pointer.StringDeref(shoot.Spec.SeedName, "") != r.SeedName {
+		return reconcile.Result{}, nil
+	}
+
+	if !shootCreatedSuccessfully(shoot.Status) {
+		log.Info("Requeuing because shoot was not yet successfully created", "requeueAfter", RequeueWhenShootIsNotYetCreated)
+		return reconcile.Result{RequeueAfter: RequeueWhenShootIsNotYetCreated}, nil
+	}
+
+	shootState := &gardencorev1beta1.ShootState{}
+	if err := r.GardenClient.Get(ctx, client.ObjectKeyFromObject(shoot), shootState); client.IgnoreNotFound(err) != nil {
+		return reconcile.Result{}, fmt.Errorf("failed fetching ShootState %s: %w", client.ObjectKeyFromObject(shoot), err)
+	}
+
+	var lastBackup time.Time
+	if v, ok := shootState.Annotations[v1beta1constants.GardenerTimestamp]; ok {
+		var err error
+		lastBackup, err = time.Parse(time.RFC3339, v)
```

Comment:
Can we convert this back to UTC after parsing, so that the `lastBackup` log field is in the same time zone as the `nextBackupDue` log field?